solutions/swb-reference/integration-tests/support/clientSession.ts

```diff
@@ -52,8 +52,11 @@ export default class ClientSession {
       },
       function (error: AxiosError) {
         if (error.response) {
+          console.error(error.response.data);
           return Promise.reject(new HttpError(error.response.status, error.response.data));
         }
+
+        console.error(error);
```

Comment:
should these log statements be here?

Reply:
Yes, this is intentional. In tests I saw only promise rejection, but no details. This helped a lot in debugging the issue